frontend/public/components/custom-resource-definition.tsx

```diff
@@ -107,6 +123,68 @@ const Established: React.FC<{ crd: CustomResourceDefinitionKind }> = ({ crd }) =
   );
 };
 
+const EmptyVersionsMsg: React.FC<{}> = () => <EmptyBox label="CRD Versions" />;
+
+const CRDVersionsHeader = [
+  {
+    title: 'Name',
+    transforms: [sortable],
+  },
+  {
+    title: 'Served',
+    transforms: [sortable],
+  },
+  {
+    title: 'Storage',
+    transforms: [sortable],
+  },
+];
+
+const CRDVersionsRows = (versions: CRDVersion[]) =>
```

Comment:
This isn't a component, so it shouldn't start with a capital letter. But I'd probably just inline this logic since it's so simple.